resources/files/windows/run_facter_interactive.bat

```diff
@@ -1,5 +1,2 @@
-@echo Running puppet on demand ...
-@echo off
-SETLOCAL

```

Comment:
Also I am realizing that I can't even fix these files until after the change in paths is done